index.html

```diff
@@ -0,0 +1,296 @@
+            <li><h1 class="year">2015</h1></li>
+          </ul>             
+        </div>
+        <h5 class="card-primary-body-text">A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </h5> 
+        <ul class="card-primary-tag">
+          <li class="tags">
+            <h2 class="html">html</h2>
+          </li>
+          <li class="tags">
+            <h2 class="html">css</h2>
+          </li>
+          <li class="java">
+            <h2 class="html">javaScript</h2>
+          </li>
+        </ul>
+        <div id="action">
+          <button type="button" class="normalbuttn"> See project</button>
+        </div>
+      </div>
+    </div>
+    <div class="card-section">
+      <div class="card-section-background-image">
+        <img src="./asset/images/professional.svg" alt="" class="card-background-image">
+      </div>
+      <div class="card-primary-text-right">
+        <div class="card-primary-text">
+          <h1 class="primary-title">Multi-post Stories</h1>
+          <ul class="card-primary-text-frame">
+            <li><h1 class="client">CANOPY</h1></li>
+            <span class="span"></span>
+            <li><h1 class="role">Back End Dev</h1></li>
+            <span class="span"></span>
+            <li><h1 class="year">2015</h1></li>
+          </ul>             
+        </div>
+        <h5 class="card-primary-body-text">A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </h5> 
+        <ul class="card-primary-tag">
+          <li class="tags">
+            <h2 class="html">html</h2>
+          </li>
+          <li class="tags">
+            <h2 class="html">css</h2>
+          </li>
+          <li class="java">
+            <h2 class="html">javaScript</h2>
+          </li>
+        </ul>
+        <div id="action">
+          <button type="button" class="normalbuttn"> See project</button>
+        </div>
+      </div>
+    </div>
+  </section> 
+  
+  <section class="about-myself-section">
+
+    <div class="abouttop">
+      <h1 class="aboutheadertext">
+        About  Myself
+      </h1>
+      <p 
+        class="aboutbodytext">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="socialmedia">
+        <h4 class="supportingText">
+          LET'S CONNECT
+        </h4>
+        <div class="frame2">
+          <ul class="icons">
+            <li><a href="twitter.com/Mc_Emarkees" class="fa-brands fa-twitter"></a></li>
+            <li><a href="github.com/emarkees" class="fa-brands fa-github"></a></li>
+            <li><a href="linkedin.com" class="fa-brands fa-linkedin-in"></a></li>
+            <li><a href="medium.com" class="fa-brands fa-medium"></a></li>
+            <li><a href="angellist.com" class="fa-brands fa-angellist"></a></li>
```

Comment:
- [optional] please kindly use a dark color for the icons like in the original figma design

Reply:
The icon's color has been changed like in the original Figma design